app/operations/story_operations/project_iteration.rb

```diff
@@ -0,0 +1,44 @@
+module StoryOperations
+  module ProjectIteration
+    def current_iteration_start
+      project_start_date + (number_of_iterations * iteration_length_in_days)
+    end
+
+    def number_of_iterations
+      (days_since_project_start / iteration_length_in_days).floor
+    end
+
+    def days_since_project_start
+      Date.current - project_start_date
+    end
+
+    def iteration_length_in_days
+      @project.iteration_length * 7
+    end
+
+    def project_start_date
+      @project.start_date
+    end
+
+    class PastIteration
+      attr_reader :start_date, :end_date, :points
+
+      def initialize(start_date, end_date, project)
```

Comment:
What do you think about using named parameters here instead of positional parameters? You could do:

```rb
def initialize(start_date:, end_date:, project:)
```

Reply:
Yes, it looks better that that. Thanks!